.packit.yaml

```diff
@@ -0,0 +1,41 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: foreman.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - foreman.spec
+    - foreman.logrotate
+    - foreman.cron.d
+    - foreman.tmpfiles
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman
+# downstream (Fedora) RPM package name
+downstream_package_name: foreman
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.spec -O foreman.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.logrotate -O foreman.logrotate"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.cron.d -O foreman.cron.d"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.tmpfiles -O foreman.tmpfiles"
+  get-current-version: "sed 's/-develop//' VERSION"
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  targets:
+    centos-stream-8:
+      additional_modules:
+      - foreman:el8
+      - ruby:2.7
+      - nodejs:17
```

Comment:
that's a typo in my PR, it needs to be 12 :woman_facepalming:

Reply:
hehe, no worries, will update it in the next round of testing